lib/datadog/core/telemetry/collector.rb

```diff
@@ -145,6 +144,7 @@ def additional_payload_variables
             format_configuration_value(configuration.tracing.writer_options[:flush_interval])
           options['logger.instance'] = configuration.logger.instance.class.to_s
           options['appsec.enabled'] = configuration.dig('appsec', 'enabled') if configuration.respond_to?('appsec')
+          options['ci.enabled'] = configuration.dig('ci', 'enabled') if configuration.respond_to?('ci')
```

Comment:
Is this still relevant @TonyCTHsu @anmarchenko? I think this might have to be moved to the datadog-ci gem instead.

Reply:
I am not sure what is the meaning of this, maybe it is important for tracer to report the value of ci.enabled to telemetry. Though it will always be false because telemetry is not enabled when CI mode is on, so you might as well remove it